VMT/templates/downstream-stakeholder-notification.md

```diff
@@ -0,0 +1,44 @@
+# Downstream stakeholders notification email (private issues) template
+
+We send two separate emails, to avoid off-topic replies to linux-distros:
+
+```
+-   *To:* <embargo-notice@lists.openstack.org>
```

Comment:
Similar to above, will need to establish this list.

Reply:
I'm adding a TODO.md file to this PR to collect up actions we need to take, including the lists we'd need to set up.